02_useR_groups_north_america.Rmd

```diff
@@ -45,7 +45,8 @@ knit: "bookdown::preview_chapter"
   * San Diego: [San Diego R Users Group](https://www.meetup.com/San-Diego-R-users-group)
   * San Diego: [University of California San Diego / Scripps Institution of Oceanography R Users Group](https://siomail.ucsd.edu/mailman/listinfo/r-users)
   * San Francisco / Bay Area: [BARUG](https://www.meetup.com/R-Users/)
-  * Santa Barbara: [Santa Barbara R Users](https://www.meetup.com/Santa-Barbara-R-Users-Group/events/107866752/)
+
```

Comment:
Would you remove this new line from your PR; it causes the next bullet point to indent.

Reply:
Sure, I am not sure how Santa Barbara line came up. I added only part from New Orleans.